ndm-operator.yaml

```diff
@@ -42,7 +42,7 @@ data:
           "name": "vendor filter",
           "state": "true",
           "include":"",
-          "exclude":""
+          "exclude":"CLOUDBYT,OpenEBS"
```

Comment:
CLOUDBYTE?

Reply:
there is a limit of 8 char for vendor names. so it will be CLOUDBYT